types/enums/ProjectTypes.ts

```diff
@@ -217,6 +219,7 @@ export const ProjectTypes = {
   'new.temporaryStructure':
     'Erect a temporary structure for historic visitor attractions and listed buildings',
   'new.warehouse': 'Storage or distribution premises',
+  unit: 'Change of units',
```

Comment:
a couple more minor project type enum changes snuck in here - can cherry pick to their own branch if noisy, but they should be quite straightforward!